tests/test_wps_update_metadata.py

```diff
@@ -31,22 +24,22 @@ def test_wps_update_metadata_opendap(opendap, updates):
     assert_response_success(resp)
```

Comment:
This is well organized and looking much simpler than before! It would be really nice if `run_wps_process` can be added to `common.py` so that it can be used throughout the entire test files, but it looks like there are many restraints to do so. Maybe it is something we should think about in the future. I think it is good to keep it this way for now.